test/performance/observed_concurency_test.go

```diff
@@ -121,7 +121,7 @@ func timeToScale(events []*event, start time.Time, desiredScale int) (time.Durat
 }
 
 func TestObservedConcurrency(t *testing.T) {
-	perfClients, err := Setup(t.Logf, true)
+	perfClients, err := Setup(t.Logf, t.Name())
```

Comment:
Similarly, do we want to disable Prometheus for this test case?

Reply:
its not used.